client/src/main/java/org/apache/celeborn/client/read/DfsPartitionReader.java

```diff
@@ -325,6 +325,11 @@ public void close() {
     } catch (IOException e) {
       logger.warn("close DFS input stream failed.", e);
     }
+    try {
+      hadoopFs.close();
```

Comment:
If fs is obtained from the cache in the client, there may be a problem with calling close here?